index.js

```diff
@@ -102,6 +102,7 @@ class Ora {
 		}
 
 		if (!this.enabled || this.isSpinning) {
+			console.log(`${logSymbols.info} ${this.text}`);
```

Comment:
It should log if `this.isSpinning` is `true`.

Reply:
Sorry not sure what you're after. Also isn't isSpinning null at that point?